examples/models/maps_cities.py

```diff
@@ -1,3 +1,12 @@
+''' A plot showing a map of the world, highlighting cities where the population is over 5,000 people, made using the 'GMapPlot' class.
```

Comment:
update for line length and code-formatting for GMapPlot
```suggestion
''' A plot showing a map of the world, highlighting cities where the population
is over 5,000 people, made using the ``GMapPlot`` class.
```

Reply:
Done.